raiden/tests/integration/test_recovery.py

```diff
@@ -50,43 +45,13 @@ def test_recovery_happy_case(
             timeout=network_wait * number_of_nodes,
         )
 
-    app0.raiden.stop()
-    host_port = (
-        app0.raiden.config['transport']['udp']['host'],
-        app0.raiden.config['transport']['udp']['port'],
-    )
-    socket = server._udp_socket(host_port)
-
-    new_transport = UDPTransport(
-        app0.discovery,
-        socket,
-        app0.raiden.transport.throttle_policy,
-        app0.raiden.config['transport']['udp'],
-    )
-
-    raiden_event_handler = RaidenEventHandler()
-    message_handler = MessageHandler()
-
-    app0_restart = App(
-        config=app0.config,
-        chain=app0.raiden.chain,
-        query_start_block=0,
-        default_registry=app0.raiden.default_registry,
-        default_secret_registry=app0.raiden.default_secret_registry,
-        transport=new_transport,
-        raiden_event_handler=raiden_event_handler,
-        message_handler=message_handler,
-        discovery=app0.raiden.discovery,
-    )
-
     app0.stop()
-    del app0  # from here on the app0_restart should be used
-
-    app0_restart.start()
+    app0.get()
```

Comment:
the change in the test seems unrelated to the addressed issue, can you add your rationale to the pr description?

Reply:
The change introduced a new argument to `App`, which required me to touch all the tests which restarted apps by using a new app instance. Since I had to touch this code, I went ahead and simplified it, by removing the old pattern and using the `Runnable` methods.